site/docs/concepts/advanced/logs-stats.md

```diff
@@ -1,5 +1,46 @@
 # Logs and statistics
 
+Flow collects logs and statistics of catalog tasks to aid in debugging and refinement of your workflows.
+
 :::caution
-Page in progress. Check back soon.
-:::
\ No newline at end of file
+Access to statistics is still a work in progress. For now, this documentation deals strictly with logs.
+:::
+
+## Logs
+
+Each organization, or tenant, that uses Flow has a `logs` collection under the global `ops` prefix.
+For the tenant Acme Co, it would have the name `ops/acmeCo/logs`.
```

Comment:
nit suggestion: For the tenant ~~Acme Co~~ `acmeCo`, it would have the name `ops/acmeCo/logs`. Otherwise it could be interpreted as implying that we somehow transform the tenant names.